### ISSUE TITLE
This is very dangerous, and the undo script doesn't work

### ISSUE THREAD

**Issue by ltrujello** [Danielshow/BoxSetting#2]
TLDR to others: **Do not trust that the current index and undo shell scripts to have been written correctly. You may seriously regret it.**

Long: Unfortunately, the undo script provided doesn't seem to work for me (as in it successfully executes but does not undo the changes). I see that it makes an attempt to save the old vim and zsh settings but that failed. I don't have time to investigate why (probably, the code is not written well and does not consider certain edge cases) but I very much want to warn others: 

**Your `.vimrc` and `.zshrc` files might be completely lost by running the shell scripts.**

Even if this does work there are also lots of options you probably don't want (ex: I don't think anyone wants a giant "DANIEL SHOW" banner in their .vimrc file, a totally unnecessary distraction on its own). Cleaning up the junk that this comes with would be a pain on its own. This really should be avoided.
